doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2385,4 +2385,17 @@ Scripts that implement this association for specific service protocols are maint
 
 ``curl -X PUT --upload-file cvoc-conf.json http://localhost:8080/api/admin/settings/:CVocConf``
 
+.. _:AllowedCurationLabels:
+
+:AllowedCurationLabels
+++++++++++++++++++++++
+ 
+A JSON Object containing lists of allowed labels (up to 32 characters, spaces allowed) that can be set, via API or UI by users with the permission to publish a dataset. The set of labels allowed 
+for datasets can be selected by a superuser - via the Dataverse collection page (Edit/General Info) or set via API call. 
+The labels in a set should correspond to the states in an organization's curation process and are intended to help users/curators track the progress of a dataset through an externally defined curation process. 
+A dataset may only have one label at a time and if a label is set, it will be removed at publication time. 
+This functionality is disabled when this setting is empty/not set.
+Each set of labels is identified by a curationLabelSet name and a JSON Array of the labels allowed in that set.
+
+``curl -X PUT -d '{"Standard Process":["Author contacted", "Privacy Review", "Awaiting paper publication", "Final Approval"], "Alternate Process":["State 1","State 2","State 3"]}' http://localhost:8080/api/admin/settings/:AllowedCurationLabels``
```

Comment:
All these API docs are great but I'm feeling like it would be nice if this feature were described at a higher level in the User Guide (the "Dataset + File Management" section, I guess). We could take a lot of the content from the release note (which is pretty ephemeral, which the guides are lasting). Then, of course, we would link to the details in the API Guide. Since this is a high-level docs thing, let me ping @djbrooke for thoughts.

Reply:
Sounds good, @qqmyers OK if I take a pass at this?

Reply:
Sure

Reply:
OK, added some additional content in https://github.com/IQSS/dataverse/pull/7967/commits/ae61cbeb53b1239978f487be34ecde69b24def23